### PR TITLE
New package: mulaRahul.Keyviz version 1.0.6

### PR DESCRIPTION
**Pull request by sitiom**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91471)

### CODE REVIEW

File: manifests/m/mulaRahul/Keyviz/1.0.6/mulaRahul.Keyviz.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: mulaRahul.Keyviz
+PackageVersion: 1.0.6
+InstallerType: inno
+Scope: user
+UpgradeBehavior: install
+ReleaseDate: 2022-08-28
+Installers:
```
Comment (stephengillie) [inline manifests/m/mulaRahul/Keyviz/1.0.6/mulaRahul.Keyviz.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Comment (stephengillie) [reply manifests/m/mulaRahul/Keyviz/1.0.6/mulaRahul.Keyviz.installer.yaml:10]: Not sure why the validation pipeline didn't catch this. The installer installs fine, but the program requires VCRedist 2015 to launch. 
image

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Version is marketed as `1.0.6` but the version written to ARP/Registry is `1.0.7` (probably a small mistake in the installer). Should be brought up to the publisher to resolve this inconsistency before we accept it into WinGet.

**Comment by mdanish-kh** [comment]
@msftbot Area-External
@msftbot Blocking-Issue

**Comment by mdanish-kh** [comment]
@msftbot reset feedback

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**